src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -1192,7 +1192,7 @@ frmelmnts.lbl.switchToJoyLayout=Switch to joyful theme
 frmelmnts.lbl.switchToOldLayout=Switch to classic theme
 frmelmnts.lbl.switchUser=Change user
 frmelmnts.lbl.tableNotAvailable=Table view is not available for this report
-frmelmnts.lbl.table = Table
+frmelmnts.lbl.table = Tables
```

Comment:
please do not change the value of a label it may be used in multiple places

Reply:
@rajeevsathish Yes, but this label was created by me exclusively for Program dashboard so it would not affect other places